test/Crowdsale.js

```diff
@@ -19,11 +22,16 @@ contract('Crowdsale', function ([_, investor, wallet, purchaser]) {
 
   const expectedTokenAmount = rate.mul(value)
 
+  before(async function() {
+    //Advance to the next block to correctly read time in the solidity "now" function interpreted by testrpc
+    await advanceBlock()
```

Comment:
Sure, I'll do my best to explain. Truffle initializes fresh blockchain context by redeploying contracts before each `contract()` block of code. Unfortunately solidity `now` function implemented by `testrpc` seems to get value from the previously mined block which is last mined block of previous `contract()` block of code. Hence, it's necessary to mine additional block only between two `contract()` groups and not between test methods.
When I've got more time I'll dig deeper wheter the fault lies on `truffle`, `testrpc` or somewhere between.